### DISCUSSION TITLE
Blank login modal on React + Vite

### DISCUSSION THREAD

**Discussion by altyni86** [Web3Auth/Web3Auth#1244] [Help]
Hi all, 

My app that runs on React + Vite is refusing to show login modal content on build and prod. staging. I have tried all the guides available, without success. What could be high probability reason for that? I am using mainnet and can see whitelisted URls... 
<img width="1440" alt="Screen Shot 2023-01-25 at 21 20 41" src="https://user-images.githubusercontent.com/95562154/214603515-b3d51144-fa91-4877-80c6-871ea529c135.png">

**Comment by VoR0220** [comment]
Check your configs closely. I ran into this on NextJS quite a few times and the answer always traced back to a faulty config.

**Reply by altyni86** [reply]
Which configs? Web3auth init? Or vite.config? What was your case?

**Reply by VoR0220** [reply]
Web3Auth init config.

**Comment by chaitanyapotti** [comment]
This is fixed in 4.4.2
